go/tasks/plugins/k8s/sagemaker/config/config.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+//go:generate pflags Config --default-var=defaultConfig
+
+import (
+	pluginsConfig "github.com/lyft/flyteplugins/go/tasks/config"
+)
+
+const sagemakerConfigSectionKey = "sagemaker"
+
+var (
+	defaultConfig = Config{
+		RoleArn: "default",
+		Region:  "us-east-1",
+		// https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-algo-docker-registry-paths.html
+		AlgorithmPrebuiltImages: map[string]map[string]map[string]string{
```

Comment:
does this work now? It was not working right

Reply:
argh... I don't think this works but I am not sure how to fix it...... @kumare3 mind giving some hint?

Reply:
I should try this, let me try.
But
``` map[string]X
where X {
map[string]Y
}```
might work

Reply:
@bnsblue I will pull your branch and try out